src/osp/UserInputHandler.h

```diff
@@ -298,6 +298,10 @@ class UserInputHandler
     void config_register_control(std::string const& name, bool holdable,
             std::initializer_list<ButtonVarConfig> vars);
```

Comment:
You do not need a version of this function that takes an initializer list.

The version that takes the std::vector is sufficent.